packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -31,6 +39,13 @@ const _SelectedItemsList = <TItem extends BaseSelectedItem>(
     });
     setItems(updatedItems);
     props.onItemsRemoved?.(itemsToRemove);
+
+    if (itemsToRemove.length === 1) {
+      const item = itemsToRemove[0];
+      onAnnounceDeletedRecipients?.(item.text);
```

Comment:
>text [](http://example.com/codeflow?start=41&length=4)

Build break, .text is not guaranteed to be on TItem.

Unsure how to address it, so posting a PR so we can more easily see the change to address the build break